braidpool_spec.md

```diff
@@ -107,19 +161,36 @@ mining process. It contains:
 
 The purpose of this data is to gather higher resolution timestamps than are
 possible if the timestamp was committed. All Braidpool timestamps are 64-bit
-fields as milliseconds since the Unix epoch. When a block header is sent to
-mining devices, many manufacturers' mining devices do not return for quite some
-time (10-60 seconds) while they compute the hash, which causes PoW-mined
-timestamps to be delayed by this amount. Adding timestamps when parents were
-seen by the node and a timestamp when the bead was broadcast allows the braid to
-compute bead times with much higher precision. Though the data is uncommitted in
-the PoW header, it is signed by a key that is committed in the PoW header, so
-third parties cannot falsify these timestamps.
+fields as milliseconds since the Unix epoch [kp]You can simply say unix
+timestamps.[/kp]. When a block header is sent to mining devices, many
+manufacturers' mining devices do not return for quite some time (10-60 seconds)
+while they compute the hash, which causes PoW-mined timestamps to be delayed by
+this amount. Adding timestamps when parents were seen by the node and a
+timestamp when the bead was broadcast allows the braid to compute bead times
+with much higher precision. Though the data is uncommitted in the PoW header, it
+is signed by a key that is committed in the PoW header, so third parties cannot
+falsify these timestamps.
+
+[kp]
+
+> Adding timestamps when parents were seen by the node and a timestamp when the
+> bead was broadcast allows the braid to compute bead times with much higher
+> precision.
+
+Why is this important? Let's explain that before we provide a solution to the
+problem.
+
+[/kp]
```

Comment:
This is described in the DAA algorithm: https://rawgit.com/mcelrath/braidcoin/master/Braid%2BExamples.html

A review of which here is needed, I agree.